tests/end_to_end/test_remote_fetch.py

```diff
@@ -42,6 +45,12 @@ def _assertLogsWritten(self):
         # client should have gotten some reads back
         self.assertEqual(len(self.client.getOutLines()), 10)
 
+    # TODO take out when #500 is fixed
+    def tearDown(self):
+        # clean up downloaded local index file
```

Comment:
Could just do a ``print("Removing index files; see issue #500")``. This will show up when we do a ``nosetests -vs`` (which is how I usually use it anyway).